hw/ip/entropy_src/dv/cov/entropy_src_UNR.vRefine

```diff
@@ -24,12 +25,15 @@
     <rule ccType="inst" ccfFlagsMask="2199136515010-42" domain="icc" entityName="entropy_src/29" entityType="block" excTime="1665673899" file="0" im-checksum="786255092" line="110" name="exclude" reviewer="unknown" text="assign intr_es_fatal_err_o          = Stub ? '0                 : core_intr_es_fatal_err;" user="1" vscope="default"></rule>
     <rule ccType="inst" ccfFlagsMask="2199136515010-42" domain="icc" entityName="entropy_src/32" entityType="block" excTime="1665673903" file="0" im-checksum="786255092" line="111" name="exclude" reviewer="unknown" text="assign alert_test                   = Stub ? stub_alert_test    : core_alert_test;" user="1" vscope="default"></rule>
     <rule ccType="inst" ccfFlagsMask="2199136515010-42" domain="icc" entityName="entropy_src/35" entityType="block" excTime="1665673907" file="0" im-checksum="786255092" line="112" name="exclude" reviewer="unknown" text="assign alert                        = Stub ? stub_alert         : core_alert;" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_param&quot;/2" entityType="toggle-bit" excTime="1665674018" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_param&quot;/1" entityType="toggle-bit" excTime="1665674018" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_param&quot;/0" entityType="toggle-bit" excTime="1665674018" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
```

Comment:
With all these manual rejections of the `--cov-unr` analysis, might it be preferable to just scrap this refine and re-run the `--cov-unr` analysis from scratch?